android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.java

```diff
@@ -251,26 +253,36 @@ public PanGestureHandler create(Context context) {
     public void configure(PanGestureHandler handler, ReadableMap config) {
       super.configure(handler, config);
       boolean hasCustomActivationCriteria = false;
-      if (config.hasKey(KEY_PAN_MIN_DELTA_X)) {
-        handler.setMinDx(PixelUtil.toPixelFromDIP(config.getDouble(KEY_PAN_MIN_DELTA_X)));
+      if(config.hasKey(KEY_PAN_ACTIVE_OFFSET_START_X)) {
+        handler.setactiveOffsetStartX(PixelUtil.toPixelFromDIP(config.getDouble(KEY_PAN_ACTIVE_OFFSET_START_X)));
```

Comment:
camel case -> `setActiveOffsetStartX` (I'd actually prefer it to be called `setActiveOffsetXStart` because "activeOffsetX" is a name of a range that has "start" and "end")